core/docker/importer/docker-compose.config.yml

```diff
@@ -2,10 +2,10 @@ version: '3.3'
 
 services:
   # container for executing config import scripts for setting up initial config
-  config-importer:
+  core-config-importer:
     container_name: core-config-importer
     build:
-      context: importer
+      context: .
       dockerfile: Dockerfile
     image: jembi/instantohie-config-importer
     restart: "no"
```

Comment:
For consistency, I think we should restart the container on failure as we do for in k8s https://github.com/openhie/instant/pull/27/files/feb9e7c6888229af6e1f9b2144984d6a97a545f8#diff-db7d6f0d86d65ba0727595cb8e75692eR40